deps/rabbit/src/rabbit_variable_queue.erl

```diff
@@ -345,6 +345,7 @@
 -define(PERSISTENT_MSG_STORE, msg_store_persistent).
 -define(TRANSIENT_MSG_STORE,  msg_store_transient).
 
+-define(INDEX, rabbit_modern_queue_index).
```

Comment:
Should we use something like `rabbit_queue_index_v2` here?

Reply:
Let's get ideas in and make a poll. :-)

Reply:
"modern" won't age well but `v2` can be easily succeeded by `v3` :)